### PR TITLE
fix: specify crowdnode worker foreground service type

### PR DESCRIPTION
**Pull request by HashEngineering**
## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [ ] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

File: integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/api/CrowdNodeWorker.kt [hunk 1]
```diff
@@ -65,7 +67,22 @@ class CrowdNodeWorker @AssistedInject constructor(
...
+                                )
+                            )
+                        } else {
+                            setForeground(
+                                ForegroundInfo(
+                                    operation.hashCode(),
+                                    notification
+                                )
+                            )
+                        }
```
Comment (HashEngineering) [inline integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/api/CrowdNodeWorker.kt:85]: This is the foreground service fix.

File: wallet/src/de/schildbach/wallet/ui/widget/ShortcutsPane.kt [hunk 2]
```diff
@@ -104,7 +104,7 @@ class ShortcutsPane(context: Context, attrs: AttributeSet) : FlexboxLayout(conte
         secureNowButton,
         explore,
         receiveButton,
-        payToContactButton,
+        if (Constants.SUPPORTS_PLATFORM) payToContactButton else payToAddressButton,
```
Comment (HashEngineering) [inline wallet/src/de/schildbach/wallet/ui/widget/ShortcutsPane.kt:107]: This is the ShortcutPane fix

File: wallet/test/de/schildbach/wallet/util/viewModels/MainViewModelTest.kt [hunk 3]
```diff
@@ -135,6 +138,10 @@ class MainViewModelTest {
         every { observe(WalletUIConfig.SELECTED_CURRENCY) } returns MutableStateFlow("USD")
     }
 
+    private val userAgentDaoMock = mockk<UserAlertDao> {
+        every { observe(any()) } returns flow { }
+    }
```
Comment (HashEngineering) [inline wallet/test/de/schildbach/wallet/util/viewModels/MainViewModelTest.kt:143]: I'm not sure why this problem started happening now and not previously on the dashpay branch.  

`observeBlockchainState_replaying_notSynced` started to fail.

**Review by Syn-McJ — APPROVED** [review]
Looks good

**Pull request merged**